src/models/Drink.py

```diff
@@ -0,0 +1,19 @@
+from bson import ObjectId
+
+class Drink:
+
+  def __init__(self, user_id: ObjectId, ingredients: str) -> None :
+    self._id: ObjectId
```

Comment:
This is dangling and should be fixed with an assignment to `self._id`. All this does right now is declare its type.